app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java

```diff
@@ -29,4 +34,19 @@ public void showDialogFragment(ListenerDialogFragment listenerDialogFragment) {
     public void showNumberOfParking(String numberOfParking) {
         binding.textViewMainActivity.setText(numberOfParking);
     }
+
+    @Override
+    public void showReservationActivity() {
+        Context context = getContext();
+        Activity activity = getActivity();
+        if (context != null && activity != null) {
```

Comment:
It's not necessary to validate context and activity, valid only one

Reply:
done